### ISSUE TITLE
[BUG] GStreamer error: clock problem

### ISSUE THREAD

**Issue by ihsanbudiman** [livekit/egress#360]
i got error when using composite egress

2023-05-12T11:55:37.635Z	INFO	egress	service/handler_deprecated.go:157	egress updated	{"nodeID": "NE_8PRF9hLNdTfA", "handlerID": "EGH_ib6SF3XSjeme", "clusterID": "", "egressID": "EG_dxAdQx6oqYQ6", "egressID": "EG_dxAdQx6oqYQ6", "request_type": "track_composite", "output_type": "file", "status": "EGRESS_ENDING"}
2023-05-12T11:55:37.711Z	ERROR	egress	pipeline/watch.go:88	GStreamer error: clock problem.	{"nodeID": "NE_8PRF9hLNdTfA", "handlerID": "EGH_ib6SF3XSjeme", "clusterID": "", "egressID": "EG_dxAdQx6oqYQ6", "element": "GstMP4Mux", "error": "Impossible to configure latency: max 0:00:04.610750000 < min 0:00:24.325000000. Add queues or other buffering elements."}
github.com/livekit/egress/pkg/pipeline.(*Pipeline).handleMessageWarning
	/workspace/pkg/pipeline/watch.go:88
github.com/livekit/egress/pkg/pipeline.(*Pipeline).messageWatch
	/workspace/pkg/pipeline/watch.go:43
github.com/tinyzimmer/go-gst/gst.goBusFunc
	/root/go/pkg/mod/github.com/livekit/go-gst@v0.2.34-0.20230210170313-8fc9f59623d4/gst/cgo_exports.go:128
_cgoexp_eb73c21f527d_goBusFunc
	_cgo_gotypes.go:17388
runtime.cgocallbackg1
	/usr/lib/go-1.18/src/runtime/cgocall.go:314
runtime.cgocallbackg
	/usr/lib/go-1.18/src/runtime/cgocall.go:233
runtime.cgocallback
	/usr/lib/go-1.18/src/runtime/asm_amd64.s:971
runtime.systemstack_switch
	/usr/lib/go-1.18/src/runtime/asm_amd64.s:436
runtime.cgocall
	/usr/lib/go-1.18/src/runtime/cgocall.go:167
github.com/tinyzimmer/go-glib/glib._Cfunc_g_main_loop_run
	_cgo_gotypes.go:2822
github.com/tinyzimmer/go-glib/glib.(*MainLoop).Run.func1
	/root/go/pkg/mod/github.com/livekit/go-glib@v0.0.0-20230223001336-834490045522/glib/gmainloop.go:53
github.com/tinyzimmer/go-glib/glib.(*MainLoop).Run
	/root/go/pkg/mod/github.com/livekit/go-glib@v0.0.0-20230223001336-834490045522/glib/gmainloop.go:53
github.com/livekit/egress/pkg/pipeline.(*Pipeline).Run
	/workspace/pkg/pipeline/pipeline.go:205
github.com/livekit/egress/pkg/service.(*HandlerV0).Run.func2
	/workspace/pkg/service/handler_deprecated.go:85

**Comment by frostbyte73** [comment]
Is this reproducible? Something in your pipeline is running with a 24 second latency, which is definitely not normal

**Comment by ihsanbudiman** [comment]
why i try use latest image error is gone, but there is something wrong with latest version i got so many logs looks like this, and cpu usage is high when logs is show

```sh
2023-05-17T09:20:54.309Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.208598813s", "calculated": "1m45.910466666s"}
2023-05-17T09:20:54.309Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.208684826s", "calculated": "1m45.904133333s"}
2023-05-17T09:20:54.309Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.208760966s", "calculated": "1m45.9458s"}
2023-05-17T09:20:54.318Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.465239143s", "calculated": "1m44.278033333s"}
2023-05-17T09:20:54.318Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.46538676s", "calculated": "1m44.2877s"}
2023-05-17T09:20:54.318Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.465446317s", "calculated": "1m44.361366666s"}
2023-05-17T09:20:54.318Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.218021344s", "calculated": "1m46.019466666s"}
2023-05-17T09:20:54.318Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.218124064s", "calculated": "1m46.029133333s"}
2023-05-17T09:20:54.318Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.218241903s", "calculated": "1m46.1028s"}
2023-05-17T09:20:54.339Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.239330196s", "calculated": "1m46.144466666s"}
2023-05-17T09:20:54.340Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.239559937s", "calculated": "1m46.154133333s"}
2023-05-17T09:20:54.340Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_DwfpJko5nbE6", "clusterID": "", "egressID": "EG_3QWPTQJKCBrq", "expected": "1m59.239616155s", "calculated": "1m46.2118s"}
2023-05-17T09:20:54.339Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.487125192s", "calculated": "1m44.403033333s"}
2023-05-17T09:20:54.340Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.487569635s", "calculated": "1m44.4127s"}
2023-05-17T09:20:54.340Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.487691805s", "calculated": "1m44.470366666s"}
2023-05-17T09:20:54.371Z	WARN	egress	synchronizer/track.go:181	timestamping issue	{"nodeID": "NE_7wJFVFcp9kPT", "handlerID": "EGH_b6qvC46wLp5i", "clusterID": "", "egressID": "EG_aRPo6qumirDD", "expected": "1m57.519018166s", "calculated": "1m44.528033333s"}
```

**Comment by frostbyte73** [comment]
That's a sign of timestamping issues, the next release is fixing that

**Comment by ihsanbudiman** [comment]
> That's a sign of timestamping issues, the next release is fixing that

tq sir 😄

**Comment by Michkail** [comment]
> That's a sign of timestamping issues, the next release is fixing that

how long will it take to prepare for the next release? because I am delighted with this feature

**Comment by frostbyte73** [comment]
@ihsanbudiman @Michkail this should be fixed in 1.7.2, please reopen if you see it again
